crates/chia-sdk-bindings/src/secp.rs

```diff
@@ -0,0 +1,116 @@
+use bindy::Result;
+use chia_protocol::{Bytes32, BytesImpl};
+use chia_secp::{
+    K1PublicKey as K1PublicKeyRs, K1SecretKey as K1SecretKeyRs, K1Signature as K1SignatureRs,
+    R1PublicKey as R1PublicKeyRs, R1SecretKey as R1SecretKeyRs, R1Signature as R1SignatureRs,
+};
+
+#[derive(Clone)]
+pub struct K1SecretKey(pub K1SecretKeyRs);
+
+impl K1SecretKey {
```

Comment:
somehow if seems there ought to be a less verbose way to solve the problem these thin wrappers solve